lib/kb_SPAdes/kb_SPAdesImpl.py

```diff
@@ -625,15 +597,15 @@ def run_SPAdes(self, ctx, params):
 
         # parse the output and save back to KBase
         output_contigs = os.path.join(spades_out, 'scaffolds.fasta')
-        if 'min_contig_len' in params and int(params['min_contig_len']) > 0:
-            self.log ("Filtering out contigs with len < min_contig_len: "+str(params['min_contig_len']))
-            output_contigs = self.filter_contigs_file (output_contigs, int(params['min_contig_len']))
+
+        min_contig_len = params['min_contig_len'] if 'min_contig_len' in params and params['min_contig_len'] > 0 else 0
```

Comment:
Slightly more compact/readable for reference:
`min_contig_len = params['min_config_len'] if params.get('min_config_len') > 0 else 0`